lib/crowdin-api/client/client.rb

```diff
@@ -67,6 +69,84 @@ def logger_enabled?
       config.logger_enabled?
     end
 
+    #
+    # FetchAll options:
+    # * limit, Integer, default: 500 | How many records need to load per one request
+    # * offset, Integer, default: 0 | How many records need to skip
+    # * request_delay, Integer (seconds), default: 0 | Delay between requests
+    #
+    #
+    # Note: Please, specify project_id while Client initialization if you need to use methods that need it within FetchAll
```

Comment:
We do not need to require the project_id here. There are some list methods on the **organization** level